echo-core/src/main/java/com/netflix/spinnaker/echo/services/Front50Service.java

```diff
@@ -1,14 +1,15 @@
 package com.netflix.spinnaker.echo.services;
 
 import com.netflix.spinnaker.echo.model.Pipeline;
+import java.util.Map;
 import retrofit.http.*;
 
 import java.util.List;
 
 public interface Front50Service {
   @GET("/pipelines?restricted=false")
   @Headers("Accept: application/json")
-  List<Pipeline> getPipelines();
+  List<Map> getPipelines(); // Return Map here so we don't throw away MPT attributes.
```

Comment:
Casting would be safer if it were `List<Map<String, Object>>`  Likewise for the code in `PipelineCache`, `Map` -> `Map<String, Object>`.

Reply:
Done.

Reply:
We've previously discussed having `@JsonAnyGetter` and `@JsonAnySetter` on `Pipeline` which is arguably preferable to falling back to it just being a `Map`? 

Believe we've sorta just pushed this off until we ran into it again.

Reply:
Looking into the JsonAny* stuff.